apps/site/lib/site_web/controllers/customer_support_controller.ex

```diff
@@ -295,10 +302,71 @@ defmodule SiteWeb.CustomerSupportController do
       service: params["service"],
       subject: params["subject"],
       no_request_response: params["no_request_response"] == "on",
-      incident_date_time: params["date_time"]
+      incident_date_time: params["date_time"],
+      mode: params["mode"],
+      line: params["route"],
+      vehicle: params["vehicle"]
     })
   end
 
+  @spec get_all_modes() :: list
+  def get_all_modes() do
```

Comment:
Since you only use this in the template I think this function would work well in `customer_support_view`, or you could leave it here and consider adding it to conn.assign via a Plug (see `set_service_options` for an example).

Reply:
Ah good point!